core/pta/bcm/sotp.c

```diff
@@ -19,6 +20,20 @@ enum pta_bcm_sotp_cmd {
 
 #define SOTP_TA_NAME		"pta_bcm_sotp.ta"
 
+static bool sotp_access_disabled;
+
+/**
+ * close_session() - Print a debug message when closing a session and set the
+ *		     driver to disallow any more pta sessions to connect.
+ * @pSessionContext	Unused.
+ */
+static void close_session(void *pSessionContext __unused)
+{
+	DMSG("close entry point for static ta \"%s\"", SOTP_TA_NAME);
```

Comment:
The name "static TA" is deprecated, they are called "pseudo TA" now, but you could as well drop "static TA" like below.